Hakyll/Convert/Common.hs

```diff
@@ -20,3 +23,13 @@ data DistilledPost = DistilledPost
     , dpDate  :: UTCTime
     }
   deriving (Data, Typeable)
+
+instance Default DistilledPost where
+  def = DistilledPost
+    { dpUri        = ""
+    , dpBody       = ""
+    , dpTitle      = Nothing
+    , dpTags       = []
+    , dpCategories = []
+    , dpDate       = fromJust $ parseTimeM True defaultTimeLocale rfc822DateFormat "1970-01-01T00:00:00Z"
```

Comment:
No `Default` instance, no `defaultUTCTime` :( But I find another, shorter and cleaner, way to achieve the same. Just pushed an updated version.